overview.md

```diff
@@ -0,0 +1,76 @@
+# HowTo: Components – Overview {: .page-title }
+{% include "web/_shared/contributors/ewagasperowicz.html" %}
+{% include "web/_shared/contributors/noms.html" %}
+{% include "web/_shared/contributors/robdodson.html" %}
+{% include "web/_shared/contributors/surma.html" %}
+
+
+"HowTo: Components" are a collection of web components that implement common UI
+patterns. The purpose of these implementations is to be an educational resource.
+You can read through the densely commented implementation of different
+components and hopefully learn from them. Note that they are explicitly **NOT**
+a UI library and should **NOT** be used in production.
+
+## Goals
+
+Our aim is to demonstrate best practices for writing robust components that are
+accessible, performant, maintainable, and easy to style. Each component is
+completely self-contained so it can serve as a reference implementation.
+
+### Accessibility
+
+The components closely follow the [WAI ARIA Authoring
+Practices](https://www.w3.org/TR/wai-aria-practices-1.1/), which is a guide to
+explain and show ARIA, the [Accessible Rich Internet Application
+standard](https://www.w3.org/TR/wai-aria-1.1/). If you are unfamiliar with ARIA,
+[check out our introduction on
+WebFundamentals](https://developers.google.com/web/fundamentals/accessibility/semantics-aria/).
+Each component links to the relevant section of the Authoring Practices. While
+not strictly necessary, we do recommend reading the section of the Authoring
+Practices before diving into the code.
+
+### Performance
+
+In web development the term "performance" can be applied to a multitude of
+things. In the context of `<howto>`, performance mostly refers to animations
+consistently running at 60fps, even on mobile devices.
+
+### Visual Flexibility
+
+As much as possible, components are not styled, except for layout or to indicate
+a selected or active state. This is to keep the implementation visually flexible
+and focused. By not spending time on decoration, we limit the code to only what
+is absolutely necessary to make the component function. If any style is required
+for the component to function, the style will be marked with a comment
+explaining why that is.
+
+### Maintainable code
+
+As `<HowTo>` is aimed to be read and function as a reference implementation, we
+spent extra time on writing readable and easily comprehensible code that is
+densely commented.
+
+## Non-Goals
+
+### Be a library / framework / toolkit
+
+`<HowTo>` components are not published on npm, bower or any other platform
+because they are not meant to be used in production. For the sake of terse,
+readable code, we are using modern JavaScript APIs and are supporting a subset
```

Comment:
maybe instead of saying "supporting a subset of browsers" we should say something like "and are supporting those browsers which implement the Web Components standards."